helm_deploy/court-hearing-event-receiver/values.yaml

```diff
@@ -1,7 +1,7 @@
 generic-service:
   nameOverride: court-hearing-event-receiver
 
-  serviceAccountName: "court-facing-api"
```

Comment:
Reverting this change which has not made it to production yet
One thing at a time